<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>668</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>481</width>
      <height>431</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTableWidget" name="tableWidget">
       <column>
        <property name="text">
         <string>Name</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Studienfach</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Jahrgang</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveFile">
       <property name="text">
        <string>Speichern</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addCell">
       <property name="text">
        <string>Zeile hinzufuegen</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSign_In">
    <property name="title">
     <string>Sign In</string>
    </property>
   </widget>
   <addaction name="menuSign_In"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
